src/game/Game.js

```diff
@@ -364,6 +362,18 @@ export class Game {
       you.ys = 0;
     }
   }
+  processOnTouch(block) {
+    // If the block being touched has an onTouch handler
+    if (block.onTouch) {
+      this.currentBlockTouch = block;
```

Comment:
I don't see `this.currentBlockTouch` being used anywhere else. Is it used in the custom `onTouch` code?

Reply:
`this.currentBlockTouch` is used to set `this.lastBlockTouch` once all the current frame's processing is done on line 374. `this.currentBlockTouch` is then set to `undefined` on line 375 so that it always gets reset, even in the wombat is not touching a block in the next frame. Let me know if there is anything I can do (code change, adding a comment, etc.) to clear this up.